src/diffusers/pipelines/stable_diffusion_self_segment_pipeline/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers
```

Comment:
We cannot add these dependencies in a requirement file, let's try to make the optional